tests/template_tests/test_response.py

```diff
@@ -222,8 +222,7 @@ def test_headers(self):
             {'value': 123, 'fn': datetime.now},
             headers={'X-Foo': 'foo'},
         )
-        response.render()
-        assert 'X-Foo' in response.headers
+        assert response.headers['X-Foo'] == 'foo'
```

Comment:
Please use `assertEqual()`  instead of `assert ... == ...`.

Reply:
Gah, sorry, force of habit, I'm too used to pytest these days.